subsys/bluetooth/host/conn.c

```diff
@@ -39,6 +39,7 @@
 #include "att_internal.h"
 #include "iso_internal.h"
 #include "direction_internal.h"
+#include "zephyr/sys/util_macro.h"
```

Comment:
```suggestion
#include <zephyr/sys/util_macro.h>
```

And move to the other `<zephyr...>` includes

Reply:
Why is this added btw? No code added seems to need it

Reply:
`IS_ENABLED`